sdks/python/apache_beam/runners/portability/fn_api_runner/translations.py

```diff
@@ -478,8 +478,10 @@ def leaf_transform_stages(
     if root.spec.urn in known_composites:
       yield Stage(root_id, [root], parent=parent)
     elif not root.subtransforms:
-      # Make sure its outputs are not a subset of its inputs.
-      if set(root.outputs.values()) - set(root.inputs.values()):
+      # Yield if its outputs are not a subset of its inputs,
+      # or if it has no inputs and no outputs.
+      if set(root.outputs.values()) - set(root.inputs.values()) or \
```

Comment:
The convention in Beam is to use ()'s rather than backslashes for multi-line statements.

Reply:
I decided to solve this in a different way, which removes this line and also fixes the test failures.